GTMAppAuth/Sources/GTMAppAuthFetcherAuthorization+Keychain.m

```diff
@@ -33,14 +40,30 @@ + (GTMAppAuthFetcherAuthorization *)authorizationFromKeychainForName:(NSString *
 }
 
 + (BOOL)removeAuthorizationFromKeychainForName:(NSString *)keychainItemName {
-  return [GTMKeychain removePasswordFromKeychainForName:keychainItemName];
+  return [GTMAppAuthFetcherAuthorization removeAuthorizationFromKeychainForName:keychainItemName
+                                                     withDataProtectionKeychain:NO];
+}
+
++ (BOOL)removeAuthorizationFromKeychainForName:(NSString *)keychainItemName
+                    withDataProtectionKeychain:(BOOL)dataProtectionKeychain {
```

Comment:
Similar comment above about the second param name not sounding truthy.

Reply:
Resolving as dupe of https://github.com/google/GTMAppAuth/pull/151#discussion_r863249309